addons/mail/web_push.py

```diff
@@ -129,8 +129,9 @@ def push_to_end_point(base_url, device, payload, vapid_private_key, vapid_public
         #  - "k" the base64url-encoded key that signed that token.
         'Authorization': 'vapid t={}, k={}'.format(token, vapid_public_key),
         'Content-Encoding': 'aes128gcm',
-        'TTL': '0',
     }
+    if 'wns2-ln2p.notify.windows' not in endpoint:
+        headers['TTL'] = '0'
```

Comment:
TTL is require by spec, I'd just make it longer than 0 for microsoft stuff if doing something special for them.

Reply:
Thanks for the review 🙂 the condition is not ideal, however I don't see what else could we use to differentiate them, let me know if you have any idea. Also added a 20s for the TTL to Edge, it seems to work well.